<launch>
  <arg name="fname" default="test"/>
  <arg name="csv_location" value="$(find controller_recorder)/csvs"/>
  <!-- Tilde operator doesn't work here -->
  <arg name="plot_location" value="$(find controller_recorder)/Plots"/>
  <param name ="/use_sim_time" value="true"/>
  
  <!-- <node name="read_csv_publisher" pkg="controller_recorder" type="read_csv_publisher.py" /> -->
  <node name="csv_twist_publisher" pkg="controller_recorder" type="csv_twist_publisher.py" />
  <node name="get_twist_subscriber" pkg="controller_recorder" type="get_twist_subscriber.py" >
    <param name="csv_save_location" type="string" value="$(arg csv_location)" />
    <param name="plot_save_location" type="string" value="$(arg plot_location)" />
    <param name="file_name" type="string" value="$(arg fname)" />
  </node>
</launch>